content/learn/book/next-steps/_index.md

```diff
@@ -8,8 +8,8 @@ page_template = "book-section.html"
 
 You have reached the end of The Bevy Book! And unfortunately, we haven't even scratched the surface of Bevy's features! Eventually this book will cover almost every facet of Bevy, but until then we recommend checking out:
 
-* **[The Bevy Examples](https://github.com/bevyengine/bevy/tree/main/examples#examples)**: We create an example for every major Bevy feature. This is currently the best way to learn Bevy's features and how to use them.
-* **[Breakout](https://github.com/bevyengine/bevy/blob/main/examples/game/breakout.rs)**: A small "breakout" clone that illustrates what a real Bevy game looks like.
+* **[The Bevy Examples](https://github.com/bevyengine/bevy/tree/latest/examples#examples)**: We create an example for every major Bevy feature. This is currently the best way to learn Bevy's features and how to use them.
```

Comment:
This link continues to use `#examples` at the end, so if https://github.com/bevyengine/bevy-website/pull/109/files#r592758938 decides to not use `#examples`, perhaps this should be aligned with the other links.